components/gsl/Makefile

```diff
@@ -9,36 +9,39 @@
 #
 
 #
-# Copyright (c) 2013 Louis M. Picciano. All rights reserved.
+# Copyright (c) 2013-2016 Louis M. Picciano. All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
+include $(WS_TOP)/make-rules/shared-macros.mk
```

Comment:
Ah... my bad - saw the question in email but did not see the context ;-\
Anyhow, the point stands - other locations welcome to use explicit double-dots just once per Makefile ;)